src/components/picker/picker.tsx

```diff
@@ -275,6 +275,7 @@ export class Picker {
                 onStopEdit={this.handleStopEditAndBlur}
                 emptyInputOnBlur={false}
                 clearAllButton={this.multiple && !this.chipSetEditMode}
+                autocomplete="nope"
```

Comment:
```suggestion
                autocomplete="off nope"
```
We should probably still use the `off` keyword

Reply:
You might want to read up on the issue 😂

Reply:
Or, well… no, you probably don't want to do that to be honest…

At least keep something hug-friendly nearby if you do. It's not exactly life-affirming reading… 😣

Reply:
Oh! Hang on, your suggestion is to _also_ use the `off` keyword. Oh, yeah, that makes sense! Sorry, just ignore me, apparently I'm having one of those days again 🙈 😅

Reply:
Why `nope` though @Kiarokh? Didn't using a random string work?

Reply:
I did read up on the issue before making the comment :smile: I also read some of those links on the bug reports. I didn't read everything in detail though so maybe this suggestion won't work, but it seems like the `autocomplete` attribute accepted a space separated list of keywords :smile:

Reply:
Also, using `nope` here, will there be any risk that it start suggesting stuff from all the places where `nope` has been used? :smile:

Reply:
> Also, using `nope` here, will there be any risk that it start suggesting stuff from all the places where `nope` has been used? 😄

Yeah, that's exactly my worry over using a static value rather than a random string. That's how I imagine I would have built the feature…

Reply:
> Why `nope` though @Kiarokh? Didn't using a random string work?

No I was just working on other stuff. I'll replace it now

Reply:
⚡